Andela-JUdensi/Inverted-Index
# Update application

#### What does this PR do?

1. Change filenames format
2. Modify lines to meet 80 specs
3. Modify app to use document titles rather than doc 1...

#### Description of Task to be completed?
1. Change multiple filenames to meet convention
2. Refactor class file to meet standard
3. Rename files to meet convention
4. Create index with document titles


#### How should this be manually tested?
1. Visit the the review app to see changes
2. View changes to the review branch

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
